### ISSUE TITLE
Display notification at the very end

### ISSUE THREAD

**Issue by soullivaneuh** [pyrech/composer-notifier#1]
With a `composer update` command on a symfony full stack project, the notification appear after packages update, but Symfony has a lot of script launched by composer.

Would be great to have the notification at the very end of the composer process command.

**Comment by pyrech** [comment]
Hi, sorry for the delay.

I just tested with a fresh symfony full stack project and the notification is triggered when the command finish. Do you mean that the notification should be triggered after the composer work and before symfony scripts? If so, I disagree because the main goal of this plugin was actually to be alerted when the whole process is finished in case you put your terminal away.

**Comment by soullivaneuh** [comment]
> Do you mean that the notification should be triggered after the composer work and before symfony scripts?

No. Actually the notification **is** triggered after composer work and not after symfony scripts.

**Comment by pyrech** [comment]
Huhu I completely forgot this issue :disappointed:
Would your issue be fixed by just changing the priority of the listener here https://github.com/pyrech/composer-notifier/blob/master/src/NotifierPlugin.php#L52?

I can't remember why did I put a negative priority. I think the greater the better in our use case.

**Comment by soullivaneuh** [comment]
I can't test right now, the concerned project are under Docker now.

I suppose you can reproduce with is by creating a SF full stack project.
